### PR TITLE
Fix Windows build

### PR DESCRIPTION
**Pull request by kozlovic**
Code referenced Debugf which is not available in this package.

/cc @derekcollison

### CODE REVIEW

File: server/pse/pse_windows.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func getStatsForProcess(name string, pcpu *float64, rss, vss *int64, pid *int) (
 			return nil
 		} else {
 			// something went wrong executing the command
-			Debugf("exec failure: %s\n", string(out))
+			// Debugf("exec failure: %s\n", string(out))
```
Comment (derekcollison) [inline server/pse/pse_windows.go]: Just remove instead of leaving the code here.

**Comment by ColinSullivan1** [comment]
Did something change?  It built and ran fine for me locally.

**Comment by kozlovic** [comment]
@ColinSullivan1 Yes, Derek moved the pse code/test into a pse subdirectory/package.

**Comment by derekcollison** [comment]
Can we remove the commented out code?

**Comment by kozlovic** [comment]
@derekcollison I did, please check again ;-)

**Comment by derekcollison** [comment]
Apologies.

**Pull request merged**